components/cam/src/physics/cam/micro_p3.F90

```diff
@@ -2814,21 +2815,17 @@ subroutine droplet_activation(t,pres,qv,qc,inv_rho,sup,xxlv,npccn,log_predictNc,
    if (log_predictNc) then
       ! for predicted Nc, use activation predicted by aerosol scheme
       ! note that this is also applied at the first time step
-      if (sup.gt.1.e-6) then
+      if (qc .gt. qsmall) then
```

Comment:
I believe that any change to a subroutine that has a C++ equivalent also needs to be changed in the C++ code.  Like in this case.  We should make a note of this, and once we have agreed on a final version of this PR go ahead and make the change in C++ as well.

Reply:
Yeah, the deal I have with Kobby is that he will get the F90 looking good and then - before merging to master - I will make the needed changes in C++.